rules-reviewed/quarkus/springboot/springboot-integration-to-quarkus.windup.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="springboot-integration-to-quarkus"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset gives hints to migrate from SpringBoot Integration to Quarkus
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-xml,3.0.0.Final" />
+        </dependencies>
+        <sourceTechnology id="springboot" />
+        <targetTechnology id="quarkus" />
+    </metadata>
+    <rules>
+        <rule id="springboot-integration-to-quarkus-00000">
+            <when>
+                <dependency groupId="org.springframework.integration" artifactId="spring-integration-core"/>
+            </when>
+            <perform>
+                <hint title="SpringBoot Integration is not supported. A migration to Apache Camel is mandatory." effort="7" category-id="mandatory">
```

Comment:
If it's in the message body then it doesn't need to be in the title too
```suggestion
                <hint title="SpringBoot Integration is not supported" effort="7" category-id="mandatory">
```

Reply:
ok